bokehjs/examples/stocks/stocks.ts

```diff
@@ -59,6 +51,8 @@ module Stocks {
     const period = 0.5;  // seconds
     function new_data() {
         t += period;
+        // XXX: explicit type annotation required due to TypeScript#6041.
+        // Remove when TypeScript 2.0 is available.
         const d: Bokeh.Data = {
```

Comment:
@almarklein, so I see where the initial confusion about strict static typing came from. As indicated in the comment, this type annotation is required due to a bug in TypeScript. When TypeScript 2.0 ships, user won't be concerned of typing details in this case, but will still get an error if he uses the wrong type.